MAiD - Case Management App/force-app/main/default/objects/Form_1634__c/fields/Length_of_assessment_period__c.field-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Length_of_assessment_period__c</fullName>
+    <externalId>false</externalId>
+    <formula>IF((NON_RFND_safeguard_completed_date__c - Clear_days_waiting_period_start_date__c)&lt;90, &apos;less than 90&apos;,
+IF((NON_RFND_safeguard_completed_date__c - Clear_days_waiting_period_start_date__c)&gt;=730, &apos;More than 2 years&apos;, 
+IF((NON_RFND_safeguard_completed_date__c - Clear_days_waiting_period_start_date__c)&gt;=365,&apos;1 to 2 years&apos;,
+IF((NON_RFND_safeguard_completed_date__c - Clear_days_waiting_period_start_date__c)&gt;=180,&apos;181-365 days&apos;,
+IF((NON_RFND_safeguard_completed_date__c - Clear_days_waiting_period_start_date__c)&gt;=120,&apos;121-180 days&apos;,&apos;90 to 120 days&apos;)))))</formula>
+    <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
```

Comment:
@RanadheerRG Return type of the formula is Text, should this be BlankAsText instead of Zero? Was this mentioned by customer that they want to see 0 if the above formula conditions are not met? If the field results in zero and the report uses this field, is customer ok with showing 0?

Reply:
As per email from the client The Customer is ok with Blank as 0